packages/bun-error/index.tsx

```diff
@@ -257,7 +255,7 @@ class FancyTypeError {
   constructor(exception: JSException) {
     this.runtimeType = exception.runtime_type || 0;
     this.runtimeTypeName = RuntimeType[this.runtimeType] || "undefined";
-    this.message = exception.message;
+    this.message = exception.message ?? '';
```

Comment:
```suggestion
    this.message = exception.message || '';
```

Reply:
Fixed in https://github.com/oven-sh/bun/pull/753/commits/f4c3f73806e63f15ddb88d6893afbdfb2381a4b1